start-aws.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+
+import boto.ec2
+
+
+# AWS region
+aws_region = 'us-west-2'
+
+# Name of security group exposing ports for Buildmaster
+security_group = 'Buildbot staging'
+
+# Key name for EC2 host
+key_name = 'hybrid-master'
+
+
+ec2_image = {
+    'Fedora-x86_64-20-20140407-sda': 'ami-cc8de6fc'
```

Comment:
Is this the latest version of something in particular or must we use this version? Where is this from?